waiter/src/waiter/scheduler.clj

```diff
@@ -73,6 +73,18 @@
 (defn make-ServiceInstance [value-map]
   (map->ServiceInstance (merge {:extra-ports [] :flags #{}} value-map)))
 
+(let [name-key :waiter/scheduler-name]
```

Comment:
Nitpick: This binding seems strange to me. In my experience, Clojure keywords are used like global constant names. If we renamed this keyword, we'd probably want to global find/replace all occurrences of the keyword anyway.

Reply:
The binding extracts the common name reference in the two functions enclosed. We do not expect users to have to use this keyword directly and do not expect any other occurrences to be replaced when this keyword is renamed.

Reply:
Does this escape into JSON anywhere? If not, then I think it would be better to use `::scheduler-name` here instead. The unit tests would help you catch if for some reason you updated one but not the other. This extra `let` binding for a keyword looks so gross...